ksqldb-engine/src/main/java/io/confluent/ksql/services/KafkaTopicClientImpl.java

```diff
@@ -168,10 +169,15 @@ public boolean isTopicExists(final String topic) {
     LOG.trace("Checking for existence of topic '{}'", topic);
     try {
       ExecutorUtil.executeWithRetries(
-          () -> adminClient.get().describeTopics(
+          () -> {
+            final DescribeTopicsResult result = adminClient.get().describeTopics(
+                ImmutableList.of(topic),
+                new DescribeTopicsOptions().includeAuthorizedOperations(true));
+            return adminClient.get().describeTopics(
```

Comment:
Do you just want to return `result` here? it seems duplicated

Reply:
Mm yeah actually I think we can just go back to returning `adminClient.get()...`, this was leftover from the local POC work but I don't think we need it. Thanks for the catch